src/Apps/W1/Quality Management/app/src/Document/QltyInspectionTestHeader.Table.al

```diff
@@ -1808,11 +1821,11 @@ table 20405 "Qlty. Inspection Test Header"
     /// <summary>
     /// OnAfterAddPicture occurs after the supplied picture instream is added to the test.
     /// </summary>
-    /// <param name="QltyInspectionTestHeader">VAR Record "Qlty. Inspection Test Header"</param>
-    /// <param name="nPictureInStream">VAR InStream</param>
-    /// <param name="PictureName">VAR Text</param>
-    /// <param name="FileExtension">VAR Text</param>
-    /// <param name="FullFileNameWithExtension">VAR Text</param>
+    /// <param name="QltyInspectionTestHeader">var Record "Qlty. Inspection Test Header"</param>
+    /// <param name="nPictureInStream">var InStream</param>
+    /// <param name="PictureName">var Text</param>
+    /// <param name="FileExtension">var Text</param>
+    /// <param name="FullFileNameWithExtension">var Text</param>
     [IntegrationEvent(false, false)]
     local procedure OnAfterAddPicture(var QltyInspectionTestHeader: Record "Qlty. Inspection Test Header"; var nPictureInStream: InStream; var PictureName: Text; var FileExtension: Text; var FullFileNameWithExtension: Text)
```

Comment:
Fix `nPictureInStream` to `PictureInStream`
Unless this is already in production, which I don't think so.